pokebot.py

```diff
@@ -189,14 +205,13 @@ def win32_signal_handler(signal_type):
 
         win32api.SetConsoleCtrlHandler(win32_signal_handler, True)
 
-    preselected_profile = None
-    debug_mode = False
-    for arg in sys.argv[1:]:
-        if arg == "--debug":
-            debug_mode = True
-        elif profile_directory_exists(arg):
-            preselected_profile = load_profile_by_name(arg)
+    parsed_args = parse_arguments()
+    preselected_profile = parsed_args.profile
+    debug_mode = parsed_args.debug
+    if preselected_profile and profile_directory_exists(preselected_profile):
+        preselected_profile = load_profile_by_name(preselected_profile)
 
+    console.print(f"Starting [bold cyan]{pokebot_name} {pokebot_version}![/]")
```

Comment:
I moved this here just so that the line isn't printed when `--help` is ran, also it kind of makes sense to display this message when the GUI is about to be initialized, but also very minor.